.github/workflows/kuksa_val_unittest.yml

```diff
@@ -2,24 +2,28 @@ name: kuksa_val_unittests
 
 on:
   push:
-    branches: [ disabled ]
-    tags:
-      - '0.*'
+    branches: [ master ]
   pull_request:
-    branches: [ disabled ]
-    
+    branches: [ master ]
+    paths:
+    - ".github/workflows/kuksa_val_unittest.yml"
+    - "kuksa-val-server/**"
+    - "kuksa_certificates/**"
+    - "proto/**"
```

Comment:
this is the same issue as in #611 not the right location to look for proto files

Reply:
fixed by #633